Dockerfile

```diff
@@ -1,9 +1,5 @@
 FROM node:6.10
 
-# phantomjs workaround
```

Comment:
note: we have to delete it in https://github.com/Wikia/mobile-wiki/pull/161/files#diff-3254677a7917c6c01f55212f86c57fbfR10 as well